pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -878,10 +878,15 @@ void run() throws PulsarAdminException {
         private boolean unload;
 
         @Parameter(names = { "--split-algorithm-name", "-san" }, description = "Algorithm name for split "
-                + "namespace bundle. Valid options are: [range_equally_divide, topic_count_equally_divide]."
-                + " Use broker side config if absent", required = false)
+                + "namespace bundle. Valid options are: [range_equally_divide, topic_count_equally_divide, "
+                + "specified_positions_divide]. Use broker side config if absent", required = false)
         private String splitAlgorithmName;
 
+        @Parameter(names = { "--split-boundaries",
```

Comment:
What's format of  list<long> to provide from cli? comma separated ？ 
I think we should add an example here?